src/integrationTest/java/uk/gov/hmcts/darts/authorisation/service/AuthorisationServiceTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.darts.authorisation.service;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.darts.authorisation.model.Permission;
+import uk.gov.hmcts.darts.authorisation.model.Role;
+import uk.gov.hmcts.darts.authorisation.model.UserState;
+import uk.gov.hmcts.darts.common.entity.SecurityGroupEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.repository.SecurityGroupRepository;
+import uk.gov.hmcts.darts.common.repository.UserAccountRepository;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.hmcts.darts.common.entity.SecurityRoleEnum.COURT_CLERK;
+import static uk.gov.hmcts.darts.common.entity.SecurityRoleEnum.COURT_MANAGER;
+import static uk.gov.hmcts.darts.common.entity.SecurityRoleEnum.JUDGE;
+
+@SpringBootTest
+@ActiveProfiles({"intTest", "h2db"})
+@TestInstance(Lifecycle.PER_CLASS)
+class AuthorisationServiceTest {
+
+    @Autowired
+    private UserAccountRepository userAccountRepository;
+
+    @Autowired
+    private SecurityGroupRepository securityGroupRepository;
+
+    @Autowired
+    private AuthorisationService authorisationService;
```

Comment:
As a more standard approach utilising the new test helpers, consider extending `IntegrationBase` which allows you to eliminate the boilerplate repository autowiring here. You may need to add `SecurityGroupRepository` (and any new repositories you've added in this PR) to `DartsDatabaseStub`.